### ISSUE TITLE
Cutting a release

### ISSUE THREAD

**Issue by JustinBeckwith** [axios/axios#1657]
Greetings folks!  There have been a few critical fixes merged into master recently.  I'd very much like to cut an 0.19 release if at all possible.  

The biggest issue at the moment is the failing IE9 test suite.  As mentioned in #1656, I think it's really worth considering dropping IE9 support.  That having been said, if someone can figure out what's up with the tests, I suppose fixing them works too :)

After that comes the release.  How are releases orchestrated  today?  Who's involved in deciding when to ship?  

@nickuraltsev @mzabriskie

**Comment by traviscrist** [comment]
This would be very helpful for those using typescript as well so that socketPath can be used without building from a hash of github master. The issue was resolved in https://github.com/axios/axios/commit/524f5bf10a29d5ebd702e0867ccd63809bfd6c74

**Comment by emilyemorehouse** [comment]
I've issued the last couple of releases and am happy to continue managing them, I've just been waiting on passing tests. As soon as that's settled, I'll cut the release.

I do have access to run local code using Sauce Labs, so I've been poking at it to see if I can get the tests to behave. My gut was that something changed with the dependencies and is causing it to hang, but I haven't found anything that works.

**Comment by axe312ger** [comment]
@emilyemorehouse thank you. Everyone able to fix Sauce Labs tests is very welcome. I can try to do it on my own as soon I find time for it :)

**Comment by RikkiGibson** [comment]
Thanks for the information and please let me know if there’s any way I can assist :)

**Comment by emilyemorehouse** [comment]
I've released 0.19.0-beta.1.

This can be installed using `npm install axios@0.19.0-beta.1` or `npm install axios@next`

I'll cut the official 0.19 release when tests are passing 🙂

**Comment by RikkiGibson** [comment]
Thank you so much!!

**Comment by axe312ger** [comment]
Looks like IE 11 is green now. Maybe just a Sauce Labs hickup with Safari?

**Comment by emilyemorehouse** [comment]
We have our green checkmark back!
<img width="443" alt="screen shot 2018-08-10 at 9 23 25 pm" src="https://user-images.githubusercontent.com/4264897/43987793-e5306290-9ce3-11e8-841e-3995c6795060.png">

I'm going to give the beta a couple of weeks before releasing 0.19 -- there's already #1718 which points out a possible regression.

**Comment by rikuayanokozy** [comment]
I'd like roll out a fix to solve #1718 next week, still need more test to release 0.19

**Comment by axe312ger** [comment]
So we got 2 customers confirming that the Axios works well in their projects, our internal tests also worked out pretty well.

I guess we can release v0.19 as non-beta? @emilyemorehouse 

Here are the customer replies:
https://github.com/contentful/contentful.js/issues/273#issuecomment-418401761
https://github.com/contentful/contentful.js/issues/272#issuecomment-418335941

**Comment by axe312ger** [comment]
(The bundle size was not affected in our case, but we updated other dependencies as well: https://bundlephobia.com/result?p=contentful@7.1.0-beta0)

**Comment by emilyemorehouse** [comment]
Yeah, I can work on getting that out. I'm okay with #1718 being listed as a breaking change and moving on as-is, provided no one else is strongly opposed.

**Comment by eilgin** [comment]
More than 2 months later, there's still no news of the 0.19 release. Is this because of https://github.com/axios/axios/pull/1831? Because it looks like Sauce labs has failed to launched the headless browsers :/

**Comment by rolledback** [comment]
Any update on a 0.19? 🙁

**Comment by emilyemorehouse** [comment]
I was just looking at why CI is now failing for everything (master was stable and is now failing with only documentation changes, which is suspect). Hoping to get that fixed and cut a release with #1831 if no one is opposed.

**Comment by rolledback** [comment]
Awesome! Thank you. 😄

**Comment by XiaoningLiu** [comment]
Any update for 0.19.0?

**Comment by azizhk** [comment]
I think this issue can be closed. (Sorry for notification, just small cleanup as I see, to avoid this library from getting abandoned.)
